server/app/survey_test.go

```diff
@@ -157,9 +159,104 @@ func TestShouldSendSurvey(t *testing.T) {
 		th.MockedPluginAPI.On("KVSetWithExpiry", "user_team_filter_cache_user_id_survey_id", mock.Anything, int64(7200)).Return(nil)
 
 		survey := &model.Survey{
-			ID:              "survey_id",
-			Status:          "in_progress",
-			ExcludedTeamIDs: []string{"team_id_1"},
+			ID:             "survey_id",
+			Status:         "in_progress",
+			FilterTeamIDs:  []string{"team_id_1"},
+			TeamFilterType: model.TeamFilterExcludeSelected,
+		}
+		should, err := th.App.ShouldSendSurvey("user_id", survey)
+		require.NoError(t, err)
+		require.False(t, should)
+	})
+
+	t.Run("should send as user is include team", func(t *testing.T) {
```

Comment:
Should we have a test that it doesn't send if it's in excluded teams? Seems that the tests below are when the teams are empty

Reply:
Because the existing behaviour was to not send survey to user in specified teams, the tests for those already exists, for example we have this existing test - `should not send as user is in a excluded team`

Reply:
Ah ok, missed those in the diff but I see them now when looking at the whole fle. Thanks for clarifying 👍